docs/releasenotes/index.rst

```diff
@@ -10,6 +10,12 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
+Next release
+============
+
+* The :ref:`dataset API <api>` added support for leases.
+  Leases prevent a dataset from being deleted or moved off a node.
```

Comment:
:question: Worth expanding the sentence? Maybe not, seeing as it's a release note, but OTOH we don't have prose anywhere explaining the concept of leases.

Even just a couple of extra words might make it a bit clearer though:

`Leases prevent a dataset from being deleted or moved off a node for as long as the lease is held.`